runtime/ddr/overrides-vm

```diff
@@ -40,7 +40,6 @@ fieldoverride.J9BuildFlags.J9VM_ARCH_S390=arch_s390
 fieldoverride.J9BuildFlags.J9VM_ENV_DATA64=env_data64
 fieldoverride.J9BuildFlags.J9VM_ENV_LITTLE_ENDIAN=env_littleEndian
 fieldoverride.J9BuildFlags.J9VM_GC_COMBINATION_SPEC=gc_combinationSpec
-fieldoverride.J9BuildFlags.J9VM_GC_COMPRESSED_POINTERS=gc_compressedPointers
```

Comment:
I don't expect a build to succeed with this change: `gc_compressedPointers` is still used at  https://github.com/eclipse/openj9/blob/master/debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/J9ObjectHelper.java#L70

Reply:
I'm not sure how this works, but does it matter that we are setting this to on in cmprssptrs.feature?
```
		<flag id="gc_compressedPointers" value="true"/>
```

Reply:
Confirmed in a local build.